### PR TITLE
Feat (api) Added endpoint to get video's information

### PR DESCRIPTION
**Pull request by Isilud**
Issue #300

### CODE REVIEW

**Review by ClerySem — APPROVED** [review]

File: docs/API.md [hunk 1]
```diff
@@ -76,6 +76,21 @@ The json will be:
...
+Route: `GET /api/v1/videos/{id}/info`
+
+Json video informations
+
+The json will be:
+
+```json
+{
+  "title": "title",
+  "uploaddate": "date",
```
Comment (RaboliotLeGris) [inline docs/API.md:91]: CamerCase ?

File: src/cmd/api/controllers/video_info_handler.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		if errors.Is(err, sql.ErrNoRows) {
+			w.WriteHeader(http.StatusNotFound)
+		} else {
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+		return
+	}
+
+	videoInfo := jsonDTO.VideoToInfoJson(video)
+	payload, err := json.Marshal(videoInfo)
```
Comment (RaboliotLeGris) [inline src/cmd/api/controllers/video_info_handler.go:64]: Maybe we can do : `json.NewEncoder(w).Ecnore(videoInfo)` and with this we remove a feat lines

File: src/cmd/api/controllers/video_info_handler.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	if err != nil {
+		log.Error("Unable to parse data struct in json ", err)
```
Comment (RaboliotLeGris) [inline src/cmd/api/controllers/video_info_handler.go:66]: parse is not really the correct word here. Serialize is maybe a bit better

File: src/cmd/api/controllers/videos_list_handler_test.go [hunk 4]
```diff
@@ -18,7 +18,6 @@ import (
 	. "github.com/Sogilis/Voogle/src/cmd/api/controllers"
 	"github.com/Sogilis/Voogle/src/cmd/api/models"
 	"github.com/Sogilis/Voogle/src/cmd/api/router"
-	. "github.com/Sogilis/Voogle/src/cmd/api/router"
```
Comment (RaboliotLeGris) [inline src/cmd/api/controllers/videos_list_handler_test.go:21]: Same, it belongs in another MR

File: src/cmd/api/dto/json/jsonDTO.go [hunk 5]
```diff
@@ -41,3 +41,19 @@ func VideoToStatusJson(video *models.Video) VideoStatus {
 
 	return videoStatus
 }
+
+// VideoInfo DTO
+
+type VideoInfo struct {
+	Title       string `json:"title" example:"amazingtitle"`
+	Upload_date string `json:"uploaddate" example:"January 2, 2006 15:04:05"`
```
Comment (RaboliotLeGris) [inline src/cmd/api/dto/json/jsonDTO.go:49]: Are we sure about this format ? Why not use a more standard date format that can be understood by the frontend ?

File: docs/API.md (hunk 1 above)
Comment (Isilud) [reply docs/API.md:91]: Camelcase was not used in the rest of the file (ex : line 61-63). Should I still correct it ?

**Review by RaboliotLeGris — APPROVED** [review]
